app/assets/javascripts/oxalis/geometries/materials/plane_material_factory.js

```diff
@@ -410,31 +411,47 @@ vec4 round(vec4 a) {
   return floor(a + 0.5);
 }
 
-
-vec4 getRgbaAtXYIndex(sampler2D textures[dataTextureCountPerLayer], float textureIdx, float textureWidth, float x, float y) {
-  vec2 accessPoint = (floor(vec2(x, y)) + 0.5) / textureWidth;
-
-  // Since WebGL 1 doesnt allow dynamic texture indexing, we use an exhaustive if-else-construct
-  // here which checks for each case individually. The else-if-branches are constructed via
-  // lodash templates.
-
-  <% if (dataTextureCountPerLayer === 1) { %>
-      // Don't use if-else when there is only one data texture anyway
-      return texture2D(textures[0], accessPoint).rgba;
-  <% } else { %>
-    if (textureIdx == 0.0) {
-      return texture2D(textures[0], accessPoint).rgba;
-    } <% _.range(1, dataTextureCountPerLayer).forEach(textureIndex => { %>
-    else if (textureIdx == <%= formatNumberAsGLSLFloat(textureIndex) %>) {
-      return texture2D(textures[<%= textureIndex %>], accessPoint).rgba;
-    }
-    <% }) %>
-    else {
-      return vec4(0.5, 0.0, 0.0, 0.0);
+// Define this function for each segmentation and color layer, since iOS cannot handle
```

Comment:
Does this PR also include the iOS fixes?

Reply:
I think, GH gets confused when a PR's base is another PR and does not update the diff accordingly. I switched the base to master and then back to the actual branch. Now the diff is correct and does not show the iPad stuff.

Reply:
Great. I was just surprised to these changes here.